README.md

```diff
@@ -18,62 +18,73 @@ A Simple and Comprehensive Vulnerability Scanner for Containers, Suitable for CI
-    - [Filter the vulnerabilities by severities](#filter-the-vulnerabilities-by-severities)
-    - [Filter the vulnerabilities by type](#filter-the-vulnerabilities-by-type)
-    - [Skip an update of vulnerability DB](#skip-update-of-vulnerability-db)
-    - [Ignore unfixed vulnerabilities](#ignore-unfixed-vulnerabilities)
-    - [Specify exit code](#specify-exit-code)
-    - [Ignore the specified vulnerabilities](#ignore-the-specified-vulnerabilities)
-    - [Clear image caches](#clear-image-caches)
-    - [Reset](#reset)
-    - [Lightweight DB](#use-lightweight-db)
-  - [Client/Server](#client--server)
-    - [Server](#server)
-    - [Client](#client)
-    - [Authentication](#authentication)
+  * [Standalone](#standalone)
+    + [Scan an image](#scan-an-image)
+    + [Scan an image file](#scan-an-image-file)
+    + [Scan an OCI image](#scan-an-oci-image)
+    + [Scan a container from inside the container](#scan-a-container-from-inside-the-container)
+    + [Scan a project including a lock file](#scan-a-project-including-a-lock-file)
+    + [Embed in Dockerfile](#embed-in-dockerfile)
+    + [Save the results as JSON](#save-the-results-as-json)
+    + [Save the results using a template](#save-the-results-using-a-template)
+    + [Filter the vulnerabilities by severities](#filter-the-vulnerabilities-by-severities)
+    + [Filter the vulnerabilities by type](#filter-the-vulnerabilities-by-type)
+    + [Skip update of vulnerability DB](#skip-update-of-vulnerability-db)
+    + [Only download vulnerability database](#only-download-vulnerability-database)
+    + [Ignore unfixed vulnerabilities](#ignore-unfixed-vulnerabilities)
+    + [Specify exit code](#specify-exit-code)
+    + [Ignore the specified vulnerabilities](#ignore-the-specified-vulnerabilities)
+    + [Specify cache directory](#specify-cache-directory)
+    + [Clear caches](#clear-caches)
+    + [Reset](#reset)
+    + [Use lightweight DB](#use-lightweight-db)
+  * [Client / Server](#client--server)
+    + [Server](#server)
+    + [Client](#client)
+    + [Authentication](#authentication)
+    + [Deprecated options](#deprecated-options)
 - [Continuous Integration (CI)](#continuous-integration-ci)
-  - [Travis CI](#travis-ci)
-  - [CircleCI](#circleci)
-  - [GitLab CI](#gitlab-ci)
-  - [Authorization for Private Docker Registry](#authorization-for-private-docker-registry)
+  * [Travis CI](#travis-ci)
+  * [CircleCI](#circleci)
+  * [GitLab CI](#gitlab-ci)
+  * [Authorization for Private Docker Registry](#authorization-for-private-docker-registry)
 - [Vulnerability Detection](#vulnerability-detection)
-  - [OS Packages](#os-packages)
-  - [Application Dependencies](#application-dependencies)
+  * [OS Packages](#os-packages)
+  * [Application Dependencies](#application-dependencies)
+  * [Image Tar format](#image-tar-format)
+  * [Data source](#data-source)
 - [Usage](#usage)
-- [Comparison with other scanners](#comparison-with-other-scanners)
```

Comment:
Did you intend to remove this section? (It would make sense to remove the Microscanner row)

Reply:
Yes, I don't follow the latest features of other scanners. So, I'm not sure this comparison is still true today. I'm thinking if we should remove the comparison section. What do you think?

Reply:
I would leave it in, we can update it if someone complains that it's no longer true. 

We could also add in links to the comparison blog posts https://boxboat.com/2020/04/24/image-scanning-tech-compared/ and https://www.a10o.net/devsecops/docker-image-security-static-analysis-tool-comparison-anchore-engine-vs-clair-vs-trivy/

Reply:
I see. I reverted TOC and added the blogs.
https://github.com/aquasecurity/trivy/pull/507/commits/c26e6314795322f3dc9fbed6928b35e90de7b522